rest/route/repo.go

```diff
@@ -277,7 +281,8 @@ func (h repoIDPatchHandler) validateBranchesForRepo(ctx context.Context, newRepo
 		githubCheckIds []string
 	}{}
 	for _, p := range mergedRepos {
-		if !p.IsEnabled() {
+		err = p.MergeWithParserProject("")
```

Comment:
This isn't actually commit queue; are we allowing overall enabled to also be defined in yaml?